recipes/libcurl/all/test_package/conanfile.py

```diff
@@ -9,11 +9,19 @@ class TestPackageConan(ConanFile):
     generators = "cmake", "cmake_find_package"
 
     def build(self):
+        if tools.cross_building(self.settings) and self.settings.os in ["iOS"]:
```

Comment:
does it fail for you? for me test package builds fine for iOS, but maybe there is a problem with CMake/conan?

Reply:
when you build binaries for iOS via xcode, you need to sign them, and than it becomes extra effort . 
also, if they are command line apps there is no way to run them anyway on iOS